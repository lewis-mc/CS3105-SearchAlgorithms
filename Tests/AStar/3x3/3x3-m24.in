3 3 3 8 2 6 0 5 4 7 1