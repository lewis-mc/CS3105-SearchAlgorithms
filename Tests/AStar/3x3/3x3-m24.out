1 24 6 4 7 6 8 2 5 1 6 8 2 3 4 2 1 5 3 1 2 4 1 2 5 6